iio/iio_axi_adc/iio_axi_adc.c

```diff
@@ -0,0 +1,525 @@
+/***************************************************************************//**
```

Comment:
there is an extra "is" in the commit message:
this is permits -> this permits
and read -> and reading